docs/Using-Fleet/MDM-commands.md

```diff
@@ -0,0 +1,75 @@
+# Commands
+
+In Fleet you can run MDM commands to take some action on your macOS hosts, like restart the host, remotely.
+
+If a host is offline when you run a command, the host will run the command the next time it comes online.
+
+## Custom commands
+
+You can run a custom commands and view a specific command's results using the `fleetctl` command-line interface.
+
+To run a custom command, we will do the following steps:
+1. Create a `.xml` with the request payload
+2. Choose a target host
+3. Run the command using `fleetctl`
+4. View our command's results using `fleetctl`
+
+### Step 1: create a `.xml` file
+
+You can run any command supported by Apple's MDM protocol as a custom command in Fleet. To see the list of possible commands, head to [Apple's Commands and Queries documentation](https://developer.apple.com/documentation/devicemanagement/commands_and_queries).
+
+> The "Erase a device" and "Lock a device" commands are only available in Fleet Premium
```

Comment:
a note: this is true for the dedicated API endpoints (that are not finished yet), but we're not screening the `xml` to prevent you from running the command if you're not premium.